setup.py

```diff
@@ -3,11 +3,11 @@
 
 
 setup(
-    name='tungsten-ci-dashboard',
+    name='tungsten-ci-acid',
     version='0.0.0',
-    url='https://github.com/tungstenfabric/tungsten-ci-dashboard',
+    url='https://github.com/tungstenfabric/tungsten-ci-acid',
     author='Tungsten Fabric',
-    description='CI dashboard for tungsten fabric repo',
+    description='CI acid for tungsten fabric repo',
```

Comment:
Rollback this change. It's out of scope and in collision with https://github.com/codilime/acid/pull/1